### ISSUE TITLE
`Alt` prop on Marker not working

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet-leaflet#62]
Passing `alt` to marker doesn’t work (it won’t be read out by the screenreader)

https://leafletjs.com/reference.html#marker-alt
